docroot/themes/custom/uids_base/uids_base.theme

```diff
@@ -183,15 +183,6 @@ function uids_base_preprocess_page(&$variables) {
     }
   }
 
-  // Hide the search form by default.
-  $variables['uiowa_search_form'] = FALSE;
-
-  if (\Drupal::service('module_handler')->moduleExists('uiowa_search')) {
-    if (\Drupal::config('uiowa_search.settings')->get('uiowa_search.display_search')) {
-      $variables['uiowa_search_form'] = \Drupal::formBuilder()->getForm('Drupal\uiowa_search\Form\SearchForm');
-    }
-  }
-
```

Comment:
Do we need to keep this in, but update it to match the page.search format instead of uiowa_search_form used in the template later? (Looking at the uiowa_search test failure.)

Reply:
Yes, I was thinking we need to add that back after the conversation this morning!